modules/ROOT/partials/configuration/filetypes.adoc

```diff
@@ -7,7 +7,7 @@ include::partial$plugins/tinydrive-filetypeslist.adoc[]
 
 For example: If the application is using {cloudfilemanager} to insert images, then set `+['image']+` in the file types array.
 
-Type : `+Array<string>+`
+*Type:* `+Array<string>+`
```

Comment:
Something which we will have to do in another Jira if one doesn't already exist to make sure the Array type is consistent between the options. The main ones I have seen are `+Array+`, `+Array<string>+` and `+String[]+`. I assume the first one is the actual one we want